### ISSUE TITLE
custom javascript on keystroke

### ISSUE THREAD

**Issue by rubaboo** [brookhong/Surfingkeys#1382]
Have some questions about using Surfingkeys with Firefox.

1. Is it possible to use key combos like Alt+Down or Ctrl+Down?
2. Please suggest a code to show the dropdown menu of the Back button.
3. Please suggest a code to show the dropdown menu of Tab Manager ("List all tabs" button).

Thank you.

**Comment by nemya9066** [comment]
Yes you can use Alt+Down or Ctrl+Down the example below show a simple message just inside the function with whatever you want to do:
mapkey('<Ctrl-ArrowDown>', 'Show Text', function() {
    Front.showPopup('Yes you can map Ctrl+Down (Press Escape to close).')
})
If you simply want to map to another key then use this:
map('<Alt-ArrowDown>', 'j')
Now you will scroll down when pressing Alt+down as if you where pressing the j key.

To show the tab manager use this:
mapkey('T', 'Choose a tab with tab manager', function() {
    Front.openOmnibar({type: "Tabs"});
})
You can change the 'T' key with another one you prefer.
About the dropdown menu of the Back button I didn't quite get what you mean by that for me the BackSpace button don't do anything. If by any chance you meant search history then map a key to 't' it what it is used by default to search bookmarks/history.

**Comment by rubaboo** [comment]
I was referring to the Firefox functionality, not SK Omnibar. Here are 2 screeshots: history Back button dropdown and Firefox-native Tab Manager. I'd like to invoke them by key instead of mouse click.

![image](https://user-images.githubusercontent.com/212368/105614361-bd1cab80-5d7d-11eb-9a41-cfe160e296f2.png)


![image](https://user-images.githubusercontent.com/212368/105614322-821a7800-5d7d-11eb-8d25-e27cc6af039f.png)
